src/Windows/R/Components/Impl/InteractiveWorkflow/Implementation/RSessionCallback.cs

```diff
@@ -105,7 +105,7 @@ public Task<PlotDeviceProperties> PlotDeviceCreate(Guid deviceId, CancellationTo
         public async Task ViewLibraryAsync(CancellationToken cancellationToken = default(CancellationToken)) {
             await _coreShell.SwitchToMainThreadAsync(cancellationToken);
             var containerFactory = _coreShell.GetService<IRPackageManagerVisualComponentContainerFactory>();
-            _workflow.Packages.GetOrCreateVisualComponent(containerFactory).Container.Show(focus: true, immediate: false);
+            (_workflow.Packages as IRPackageManagerVisual).GetOrCreateVisualComponent(containerFactory).Container.Show(focus: true, immediate: false);
```

Comment:
So this is simpler split, unblocks further work. Theses areas are stable and there is no immediate work planned. There is only a week left and we should not be spilling refactoring into May. Casting is not that of a big deal, it basically same concept as `QueryInterface` for higher order interface when getting base.